com.playeveryware.eos/Runtime/Core/EnumeratorWrapper.cs

```diff
@@ -1,42 +0,0 @@
-/*
```

Comment:
I might be mistaken, but this file delete doesn't seem relevant to the namespace cleanup.

Reply:
Good call. I restored it.